src/styles/settings/calendar.scss

```diff
@@ -0,0 +1,58 @@
+$day-size: 100px;
+$calendar-padding: 82px;
+$calendar-size: ($day-size * 7) + (2 * $calendar-padding) + 6;
```

Comment:
The calculation for `$calendar-size` includes a `+ 6` at the end. Ensure this is intentional and necessary for your layout, as it might not be clear why this specific value is added.